src/createRoutine.js

```diff
@@ -1,35 +1,15 @@
-import { createAction } from 'redux-actions';
 import routineStages from './routineStages';
-
-const isFunction = value => typeof value === 'function';
+import createActionCreator from './createActionCreator';
 
 export default function createRoutine(typePrefix, payloadCreator, metaCreator) {
```

Comment:
In my example `createRoutine` was build on top of `createRoutineCreator` to reduce code duplicates

Reply:
I don't know why it's now marked as `outdated` https://github.com/afitiskin/redux-saga-routines/pull/59/files#diff-9212afb16a80781533a3d2d29a222017

Reply:
I fixed that